dashboardwm.py

```diff
@@ -0,0 +1,130 @@
+import streamlit as st
+from fortepyan import MidiPiece
+from datasets import load_dataset
+
+import utils as U
+
+
+# Define the number of records to display per page
+n_records_per_page = 8
+
+# Global variables to store filter options
+selected_composer = None
+search_by_title = None
+filter_option = None
+
+
+def load_data(selected_split):
+    # Load the dataset for the selected split
+    dataset = load_dataset("roszcz/maestro-v1-sustain", split=selected_split)
+
+    # Apply the filtering based on the current filter option
+    if filter_option == "composer" and selected_composer:
+        dataset = dataset.filter(lambda example: example["composer"] == selected_composer)
+    elif filter_option == "title" and search_by_title:
+        dataset = dataset.filter(lambda example: str(search_by_title).lower() in example["title"].lower())
+
+    return dataset
+
+
+def get_composers(dataset):
+    # Get unique composer names from the dataset
+    composers = [x for x in dataset.unique("composer")]
+    return composers
+
+def get_titles_by_composer(dataset, selected_composer):
+    # Get titles of pieces by the selected composer
+    title = [record["title"] for record in dataset if record["composer"] == selected_composer]
```

Comment:
```suggestion
    titles = [record["title"] for record in dataset if record["composer"] == selected_composer]
    titles = np.unique(titles)
```

We need titles of all the pieces available for the selected composer, but there's more than one recording of a piece, we should have it appear only once in the selection box. 

Right now it shows multiple times:

<img width="420" alt="image" src="https://github.com/Nospoko/maestro-midi-streamlit/assets/8056825/5d65ddad-6e58-4d9d-b267-14aece2adcb0">